src/modules/comments.js

```diff
@@ -1,27 +1,6 @@
-/* eslint-disable consistent-return */
 import { COMMENT_API_URL } from './config.js';
 
 class Comment {
```

Comment:
- Hey, The comment doesn't seem to have any counter. The comments are cleared whereas, it's supposed to keep track of the number of comments on each item after a user adds a comment. Please, try fixing this.

Reply:
# TESTS

- Hey, the test in the `counter.test.js` are failing. please, fix this
- Also. you should have more test that just one. please kindly update this also.

Reply:
# README 

- Please, kindly ensure you've deployed your app using GitHub pages or Netlify to meet the project requirement and to make your readme more professional.
- Please, kindly add a video description using zoom for recording and the length shouldn't be more than 5 min long to meet the project requirement.
- Let's also add some commands on how to run the test in the app to make the reame more professional.